server/core/src/https/mod.rs

```diff
@@ -208,7 +208,8 @@ pub async fn create_https_server(
                 .route("/", get(|| async { Redirect::temporary("/ui") }))
                 .route("/manifest.webmanifest", get(manifest::manifest))
                 .nest("/ui", spa_router)
-                .layer(middleware::compression::new()) // TODO: this needs to be configured properly
+                .layer(middleware::compression::new())
+                .route("/ui/images/oauth2/:rs_name", get(oauth2::oauth2_image_get))
```

Comment:
Should this be in ui? Or should it just be a get on /v1/oauth2/:rs_name/image?

Reply:
Is this route cacheable?

Reply:
I... don't think we do the caching thing properly at the moment >.>

And it's a UI-related thing more than an API thing because it doesn't respond with a JSON blob - and it's only user-facing-UI related.